docs/attributes-registry/jsruntime.md

```diff
@@ -0,0 +1,62 @@
+| `max`    | Event loop maximum latency.            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `mean`   | Event loop mean latency.               | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `stddev` | Event loop standard deviation latency. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `p50`    | Event loop 50 percentile latency.      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `p90`    | Event loop 90 percentile latency.      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `p99`    | Event loop 99 percentile latency.      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`jsruntime.gc.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value         | Description                              | Stability                                                        |
+| ------------- | ---------------------------------------- | ---------------------------------------------------------------- |
+| `major`       | Major (Mark Sweep Compact).              | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `minor`       | Minor (Scavenge).                        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `incremental` | Incremental (Incremental Marking).       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `weakcb`      | Weak Callbacks (Process Weak Callbacks). | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`jsruntime.heap.size.state` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value   | Description             | Stability                                                        |
+| ------- | ----------------------- | ---------------------------------------------------------------- |
+| `total` | Total heap memory size. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `used`  | Used heap memory size.  | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`jsruntime.heap.space.state` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value   | Description              | Stability                                                        |
+| ------- | ------------------------ | ---------------------------------------------------------------- |
+| `total` | Total heap memory space. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `used`  | Used heap memory space.  | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
i think need to add available

Reply:
can't you just get that from `total - used`?

Reply:
I believe they use used + free for memory, no total

Reply:
> can't you just get that from `total - used`?

I don't think so, in [doc](https://nodejs.org/api/v8.html#v8getheapstatistics) if count total - used (space_size - space_used_size) example it's different than space_available_size

Reply:
Changed to separated metrics of the different types